docs/docs/static-folder.md

```diff
@@ -33,23 +33,23 @@ render() {
   lessonTitle="Use a local image from the static folder in a Gatsby component"
```

Comment:
Talvez seja interessante traduzir aqui também

Reply:
Esse não, é o nome do vídeo no [egghead](https://egghead.io/lessons/gatsby-use-a-local-image-from-the-static-folder-in-a-gatsby-component) que não tem tradução em português :/